tests/Aspire.Hosting.Containers.Tests/ContainerResourceTests.cs

```diff
@@ -11,6 +11,8 @@ namespace Aspire.Hosting.Containers.Tests;
 
 public class ContainerResourceTests
 {
+    //TODO: AddContainer tests here
```

Comment:
```suggestion
```

Reply:
Oops, was a note to myself.  Thanks for catching.